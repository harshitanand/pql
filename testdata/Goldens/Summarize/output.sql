WITH "subquery0" AS (SELECT "Directory" AS "Directory", endsWith("FileName", '_test.go') AS "IsTest", sum("LineCount") AS "TotalLines" FROM "SourceFiles" GROUP BY "Directory", endsWith("FileName", '_test.go'))
SELECT * FROM "subquery0" ORDER BY "Directory" ASC NULLS FIRST, "IsTest" ASC NULLS FIRST;
