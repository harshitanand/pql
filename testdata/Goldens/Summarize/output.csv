Directory,IsTest,TotalLines
.,0,477
.,1,284
parser,0,1942
parser,1,1588
